### DISCUSSION TITLE
Converting to 'Trial-Use' Standard

### DISCUSSION THREAD

**Discussion by jimschoening1** [CommonCoreOntology/CommonCoreOntologies#380] [Ontology Terms and Class Schema]
Converting to 'Trial-Use' Standard

I'm thinking we should convert P3195.1, .1.1, and .1.2 to 'Trial-Use' standards projects. (Note: P3195 Requirements for a Mid-level Ontology is mature enough to pursue a full-use standard.)

Here are the problems I see and how this helps:

1. Problem: For CCO and Cyber, half of the 4-year schedule is gone, and we signed up for 'next month' to have an initial draft ready for balloting.  We can miss these deadlines and get extensions, but even an extra 2 years might not be enough to get mature and stable ontologies.

Solution: Switching to Trial-Use lowers the bar for the maturity of our ontologies, plus it restarts the clock so we don't have to grovel for extensions. Once approved, we get another 3-years to work on a full-use draft.

2. Problem: Our ontologies need bottom-up feedback  based on trial-use.
Solution: Approval as an IEEE Trial-Use standard should attract more attention and help get us this feedback. 

See IEEE Manual para. 5.7 on Trial-Use standards at https://standards.ieee.org/wp-content/uploads/import/documents/other/sb_om.pdf

Each subgroup should first address this idea, then we could vote on it at the OSWG level.

**Comment by johnbeve** [comment]
@jimschoening1 I believe your proposal/tasking are better suited to the discussion section of the repository; I hope you don't mind if I convert this issue then to a discussion. 

In pursuit of such discussion, I would like to suggest the CCO standards document be excluded from your proposal. 

- **Regarding 1**: @APCox and I have planned to first complete the MLO standard document then turn attention to CCO proper. We are moving along rapidly towards completion of the MLO standard document, and so anticipate focusing on the CCO document in the near future. Once we've turned our attention to the CCO document, I anticipate making rapid progress there as well.  I would be _very_ surprised if we needed 2 years to complete the CCO document. 
- **Regarding 2**: CCO regularly receives bottom-up feedback from numerous stakeholders, as evidenced by the [bustling issue tracker](https://github.com/CommonCoreOntology/CommonCoreOntologies/issues), the well-attended biweekly [BFO-CCO Office Hours](https://ncor-organization.github.io/NCOR-Test/office-hours/), numerous publications [directly and indirectly engaging with CCO](https://scholar.google.com/scholar?start=0&q=%22common+core+ontologies%22&hl=en&as_sdt=0,44), the growing list of open-source projects using CCO found on the [CCO website](https://commoncoreontology.github.io/cco-webpage/users/), as well as stakeholders using CCO in closed spaces.
